beancount2ledger/cli.py

```diff
@@ -68,11 +71,24 @@ def cli():
         action="version",
         version=f"%(prog)s {beancount2ledger.__version__}")
 
-    args = parser.parse_args()
-    config = get_config(args.config)
-    output = beancount2ledger.convert_file(
-        args.file.name, args.format, config=config)
-    print(output)
+    with contextlib.ExitStack() as stack:
+        args = parser.parse_args()
+        in_file = args.file
+
+        # beancount loader does not accept file objects, hence to support reading from
+        # stdin we write its content to a tempfile and call the loader on its path
+        if in_file == "-":
+            tmpfile = stack.enter_context(
+                NamedTemporaryFile(prefix="tmp.beancount2ledger.",
```

Comment:
I haven't tested it, but I suspect this does not work on Windows. There it is not possible to use the name of the NamedTemporaryFile to re-open the file for writing (see for the example the beancount tests failures on Windows). My understanding is that this is because NamedTemporaryFile keeps the file descriptor open and Windows does not allow to re-open the file in this case. Closing the file descriptor causes the NamedTemporaryFile to be deleted.